assets/js/components/legacy-setup/SetupUsingGCP.js

```diff
@@ -389,6 +411,8 @@ export default compose(
 					notification: 'authentication_success',
 				}
 			),
+			isUsingProxy: select( CORE_SITE ).isUsingProxy(),
```

Comment:
This component is only every used, as I understood things, when the proxy is _not_ in use, so there shouldn't be a need to use this selector (or the subsequent ternary logic in the component) here.

If this component is being used we're definitely **not** using the proxy, so we can ignore these checks.

Reply:
Hmm... This might be one for @asvinb to advise on as he wrote the IB 🤔 . I'm happy to admit this falls outside of my knowledge.
I'll make the adjustments you suggest, but it might be worth checking with Asvin in due course if there was a reason why the IB suggested this? 👍